x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -259,17 +257,31 @@ export class FleetPlugin
 
     core.http.registerRouteHandlerContext<FleetRequestHandlerContext, 'fleet'>(
       'fleet',
```

Comment:
I was wondering if `PLUGIN_ID` should have been used here as well. I was not sure, so I did not change it.